src/screens/Voucher/VoucherDetailScreen.tsx

```diff
@@ -65,6 +66,22 @@ export const VoucherDetailScreen = ({ route }: StackScreenProps<any>) => {
     title
   } = data[QUERY_TYPES.GENERIC_ITEM];
 
+  const dateOfAvailabilityText = () => {
+    const dateStart = dates?.[0]?.dateStart && momentFormat(dates[0].dateStart);
+    const dateEnd = dates?.[0]?.dateEnd && momentFormat(dates[0].dateEnd);
+
+    if (!dateStart && !dateEnd) {
+      return;
+    }
+
+    if (dateStart && dateEnd) {
+      return `${texts.voucher.detailScreen.available}: ${dateStart}-${dateEnd}`;
+    } else if (dateStart) {
+      return `${texts.voucher.detailScreen.availableFrom}: ${dateStart}`;
+    } else if (dateEnd) {
+      return `${texts.voucher.detailScreen.availableTo}: ${dateEnd}`;
+    }
+  };
```

Comment:
please move this in the `dateTimeHelper`

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1109/commits/29b4b3ab6fb78dc62194fe5c48a3420de8c8c967